bin/pyenv-installer

```diff
@@ -29,7 +29,7 @@ failed_checkout() {
 }
 
 checkout() {
-  [ -d "$2" ] || git clone --depth 1 "$1" "$2" || failed_checkout "$1"
+  [ -d "$2" ] || git -c advice.detachedHead=0 clone --branch "$3" --depth 1 "$1" "$2" || failed_checkout "$1"
```

Comment:
AFAIK, `--branch` only accepts tip names, not arbitrary commits?

Reply:
Yea, it seems that option only works with branch names or tags. I don't have a particular need to use arbitrary commits, though I notice someone else does (referring to #131)?